jobs/FunctionTest/prepare_manifest.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash -ex
-REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog")
-
+if (echo ${REPOS_UNDER_TEST} | grep -q  "image-service"); then
+    REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog" "image-service")
+else
+    REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog")
+fi
```

Comment:
echo a list will only echo the first element.

Please ```echo ${a[@]}```

However, this comments is not necessary~